galaxykit/command.py

```diff
@@ -292,7 +332,9 @@ def main():
                 (url,) = args.rest
                 print(json.dumps(client.get(url)))
             elif args.operation == "post":
-                raise NotImplementedError
+                url = args.rest[0]
+                body = ""# sys.stdin.read()
```

Comment:
maybe in the future we can have this take a file or something? not sure the best/most flexible way to take in the input for a POST request but it is probably something that would be handy to have.